ros/src/driveworks/packages/autoware_driveworks_gmsl_interface/CMakeLists.txt

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+
+cmake_minimum_required(VERSION 2.8.3)
+project(gmsl_camera)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error -Wno-deprecated-declarations -std=c++11")
+
+##################
+## ROS packages ##
+##################
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
```

Comment:
It's become a convention to add 'autoware_build_flags' as the first package.

Reply:
Also add `autoware_build_flags` to https://github.com/CPFL/Autoware/pull/1451/files/429d11eee4af615e9f552f832396ccd9aa1bf268#diff-10c9d8f1b5e5f2f7e4231066d3092b5c as a `buildtool_depend` so that catkin can find it. See:

https://github.com/CPFL/Autoware/wiki/Quality-Control#common-build-flags